.github/workflows/build.yml

```diff
@@ -12,13 +12,17 @@ jobs:
             vcpkg_path: C:\mixxx-vcpkg
             vcpkg_bootstrap: .\bootstrap-vcpkg.bat
             vcpkg_triplet: x64-windows
+            vcpkg_overlay_ports: overlay/x64-windows;overlay/ports
+            vcpkg_packages_extras: libid3tag libmad qt5-winextras
             check_disk_space: Get-PSDrive
-            #- os: macos-10.15
-            #  vcpkg_path: /Users/runner/mixxx-vcpkg
-            #  vcpkg_bootstrap: ./bootstrap-vcpkg.sh
-            #  vcpkg_triplet: x64-osx
-            #  vcpkg_cache: /Users/runner/.cache/vcpkg/archives
-            #  check_disk_space: df -h
+          - os: macos-10.15
+            vcpkg_path: /Users/runner/mixxx-vcpkg
+            vcpkg_bootstrap: ./bootstrap-vcpkg.sh
+            vcpkg_triplet: x64-osx
+            vcpkg_overlay_ports: overlay/x64-osx:overlay/ports
```

Comment:
Let's keep the CPU architecture out of the overlay path and use just the OS. Otherwise it will be awkward in the future when we make ARM builds for macOS.

Reply:
OK